src/Entities/PaymentInstrumentValidation.php

```diff
@@ -14,7 +14,7 @@
 use Rebilly\Rest\Resource;
 use DomainException;
 
-abstract class PaymentInstrumentValidation extends Resource
```

Comment:
why do you remove `abstract`?

Reply:
can't use `abstract` here: https://github.com/Rebilly/rebilly-php/pull/274/files#diff-ef8c85a3bd0e46de45694063177bf787R336

Reply:
ohh, then collection items will be incorrect :(